.evergreen/config.yml

```diff
@@ -236,7 +200,7 @@ functions:
     - command: archive.targz_pack
       params:
         target: "drivers-dir.tar.gz"
-        source_dir: ${DRIVERS_TOOLS}
```

Comment:
Perhaps we should explicitly make DRIVERS_TOOLS and PROJECT_DIRECTORY into expansions?

Reply:
I had hoped we could remove the expansions.update call entirely, but I agree here.